### PR TITLE
add homepage screens

### PR DESCRIPTION
**Pull request by PogudoTanya**

### CODE REVIEW

**Comment by apsevkin** [comment]
Screen Shot 2021-01-13 at 13 26 16
add padding

**Comment by apsevkin** [comment]
Screen Shot 2021-01-13 at 13 25 36
change status bar background color

**Comment by apsevkin** [comment]
Screen Shot 2021-01-13 at 14 34 22

File: package.json [hunk 1]
```diff
@@ -31,18 +31,21 @@
     "react-native-config": "1.4.1",
     "react-native-fbsdk": "3.0.0",
     "react-native-gesture-handler": "1.9.0",
+    "react-native-parallax-header": "1.1.4",
```
Comment (liubaterehova) [inline package.json:34]: u don't use it

File: package.json [hunk 2]
```diff
@@ -31,18 +31,21 @@
...
     "react-native-phone-number-input": "2.0.0",
     "react-native-reanimated": "1.13.2",
     "react-native-safe-area-context": "3.1.9",
     "react-native-screens": "2.15.0",
     "react-native-sensitive-info": "6.0.0-alpha.8",
     "react-native-smooth-pincode-input": "1.0.9",
     "react-native-splash-screen": "3.2.0",
+    "react-native-sticky-parallax-header": "0.4.0",
```
Comment (liubaterehova) [inline package.json:42]: u don't use it

**Review by liubaterehova — CHANGES_REQUESTED** [review]

**Pull request closed without merging**